docs/getting-started/edfi-exchange/how-to-contribute.md

```diff
@@ -0,0 +1,43 @@
+# How to Contribute
+
+## How to contribute
+
+If your organization has developed Ed-Fi aligned technologies or informational resources as part of an Ed-Fi technology implementation, please start by [creating a ticket](https://tracker.ed-fi.org/secure/CreateIssue!default.jspa) in the [Ed-Fi Tracker  project](https://tracker.ed-fi.org/projects/EDFI) (using the "Exchange Contribution" issue type) , and provide the following information in the ticket:
```

Comment:
Change to "create a case in the [Ed-Fi Community Hub](https://community.ed-fi.org), providing the following information:"